samples/matter/lock/README.rst

```diff
@@ -560,119 +560,40 @@ Upgrading the device firmware
-         * ``GetYearDaySchedule``
-         * ``GetYearDayScheduleResponse``
-         * ``ClearYearDaySchedule``
+.. include:: /includes/sample_snippets.txt
 
-      * For the ``Holiday`` schedule support:
+The following snippet is available:
 
-         * ``SetHolidaySchedule``
-         * ``GetHolidaySchedule``
-         * ``GetHolidayScheduleResponse``
-         * ``ClearHolidaySchedule``
+* ``schedules`` - Enables schedule timed access features.
 
-#. In the **Door Lock** context window, go to the **Attributes** tab and set the proper bits for the ``FeatureMap`` attribute:
+  .. note::
+    This snippet requires a custom ZAP configuration, located in the :file:`snippets/schedules` directory.
+    Ensure to configure and generate the correct ZAP files if a custom data model is needed.
+    To do this, run the ``west zap-generate`` command and choose the :file:`snippets/schedules/lock.zap` file.
 
-   * For the ``Week-day`` schedule support set the ``5th`` bit of the feature map bit map.
-   * For the ``Year-day`` schedule support set the ``11th`` bit of the feature map bit map.
-   * For the ``Holiday`` schedule support set the ``12th`` bit of the feature map bit map.
 
-#. Enable Time Synchronization cluster with all needed types in the ZAP file:
-
-   #. In ZAP Tool GUI, select the endpoint 0 and enable the ``Time Synchronization`` cluster, with both server and client roles, for that endpoint.
-
-   #. Click the :guilabel:`Configure` symbol for the ``Time Synchronization`` cluster entry.
-   #. In the **Time Synchronization** context window, go to the **Attributes** tab and enable all required attributes:
-
-      * ``UTCTime``.
-      * ``Granularity``.
-      * ``TimeZone``.
-      * ``DSTOffset``.
-      * ``LocalTime``.
-      * ``TimeZoneDatabase``.
-      * ``TimeZoneListMaxSize``.
-      * ``DSTOffsetListMaxSize``.
-      * ``TrustedTimeSource``.
-
-   #. In the **Time Synchronization** context window, go to the **Attribute Reporting** page and enable all available attribute reporting entries.
-
-   #. In the **Time Synchronization** context window, go to the **Commands** page and enable all required command entries:
-
-      * ``SetUTCTime``.
-      * ``SetTimeZone``.
-      * ``SetTimeZoneResponse``.
-      * ``SetDSTOffset``.
-      * ``SetTrustedTimeSource``.
-
-#. In the **Time Synchronization** context window, go to the **Attributes** tab and set the proper bits for the ``FeatureMap`` attribute:
-
-   * For the ``TimeZone`` support, set the ``0th`` bit of the feature map bit map.
-   * For the ``TimeSyncClient`` support, set the ``3rd`` bit of the feature map bit map.
-
-   As a result, the default decimal value of the ``FeatureMap`` should be `9`.
-
-#. Save the :file:`lock.zap` file, and close ZAP-tool.
-#. Generate new ZAP files with the changes in the Door Lock cluster using the following west command:
+.. _matter_lock_enabling_scheduled_timed_access:
 
-   .. code-block:: console
+Enabling scheduled timed access
```

Comment:
I don't think this section is needed anymore, seeing as enabling the snippet is so straightforward.

Just move the information after the example command to the section describing the feature, so no information is lost.